projects/lights_control/src/lights_gpio_config.c

```diff
@@ -0,0 +1,22 @@
+#include "misc.h"
+
+#include "gpio.h"
+#include "lights_events.h"
+#include "lights_gpio_config.h"
+
+LightsConfig *lights_config_front_load(void);
```

Comment:
These should be in headers. You can move your enums and maybe your bitsets into it as well.